src/components/CustomButton.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button, useTheme } from '@mui/material';
+
+export const CustomButton = ({children, ...props}) => {
```

Comment:
Why is this a `CustomButton`? And what is it doing?

Reply:
@silksil I noticed that you styled the button to use in the `searchForm`. 
I copied the styles from there and used in Facilities` buttons. 
I had to copy and paste these styles each time I wanted to use a button, hence why I separated it
Although, I feel the better option would be to override the style at the theme level, so that I can use the button directly

Reply:
3 buttons in the `SearchForm` are kind of a an exception to the rule wen it comes to styling: they have to have the same style as the input field when it comes to size. I could have documented this better, my fault. 

For `normal` buttons you can simply use the default sizes of the `Button` out of `mui`.